lib/puppet/provider/rabbitmq_cli.rb

```diff
@@ -25,7 +25,7 @@ def self.append_to_path(dir)
 
   def self.home_tmp_command(name, path)
     has_command name, path do
-      environment HOME: '/tmp'
+      environment HOME: '/tmp', LANG: 'en_US.UTF-8', LC_ALL: 'en_US.UTF-8'
```

Comment:
IIRC, this should already be set for most use cases via
https://github.com/voxpupuli/puppet-rabbitmq/blob/dfd7607753b1fe4a4ad1de41eb31183d1a23dca8/data/common.yaml#L82-L83

I'm not 100% sure that that applies to commands run in the provider, but I _think_ it should; I know there have been some previous fixes around this error.

Since `environment_variables` is a parameter already, maybe the right fix here is to somehow either add `HOME` to defaults or merge the defaults with the environment for the CLI here?

Either way, I think the environment fix here should probably be a separate PR, as it's a narrower fix that we should be able to merge quickly, and likely affects other OSes besides RHEL.

Did you confirm that both `LANG` and `LC_ALL` were necessary? `LC_ALL` has worked in the past and is the somewhat narrower fix.